TeamCode/src/main/java/org/firstinspires/ftc/teamcode/DriveTrain.java

```diff
@@ -0,0 +1,231 @@
+package org.firstinspires.ftc.teamcode;
+
+import android.util.Log;
+
+import com.qualcomm.robotcore.eventloop.opmode.LinearOpMode;
+import com.qualcomm.robotcore.hardware.Gamepad;
+
+import org.firstinspires.ftc.robotcore.external.Telemetry;
+import org.firstinspires.ftc.robotcore.external.navigation.AngleUnit;
+import org.firstinspires.ftc.robotcore.external.navigation.AxesOrder;
+import org.firstinspires.ftc.robotcore.external.navigation.AxesReference;
+import org.firstinspires.ftc.robotcore.external.navigation.Orientation;
+
+public class DriveTrain {
+    String DRIVE_TRAIN_CAPTION = "Drive Status";
+    Telemetry telemetry;
+    HardwareInnov8Robot robot;
+    LinearOpMode opMode;
+
+
+    double wheelOnePower = 0.0;
+    double wheelTwoPower = 0.0;
+    double wheelThreePower = 0.0;
+    double wheelFourPower = 0.0;
+    double wheelSpeedChanger = 1.1;
```

Comment:
I don't think we actually use/need wheelSpeedChanger cuz it's not being used anywhere in the file and we've never really used the speed up/ slow down features

Reply:
yeah you're right, consider it deleted